catalog/MDCCatalog/MDCCatalogComponentsController.swift

```diff
@@ -347,14 +351,19 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
                       containerView: UIView,
                       insets: UIEdgeInsets,
                       height: CGFloat) {
-    _ = NSLayoutConstraint(
-      item: label,
-      attribute: .leading,
-      relatedBy: .equal,
-      toItem: containerView,
-      attribute: .leading,
-      multiplier: 1.0,
-      constant: insets.left).isActive = true
+    if let _ = titleLabelLeadingConstraint {
```

Comment:
why not have it unwrap in the `let` and use it inside the clause without the forced unwrapping `!`

Reply:
I'm not sure - is that better?

Reply:
it's better practice to use the proper swift unwrapping rather than explicitly unwrapping. We should refrain from using `!` unless we have to.

Reply:
Ok. I was trying to avoid creating duplicated symbols. I will change it.

Reply:
you can also call it the exact same name `if let titleLabelLeadingConstraint = titleLabelLeadingConstraint {` to avoid having more variables

Reply:
Isn't that a separate variable within the new scope that has the same name as the instance variable?

Reply:
correct, it will be scoped to use the newly created variable, but then it would cause less naming confusion. up to your own preference

Reply:
Have a look here: https://github.com/raywenderlich/swift-style-guide#optionals
Not our own style guide, but i agree on the naming.

Reply:
Ok, that makes sense to me.  I used a different name this time. I've found duplicated variable names across scopes (and recommendations against using `self`) contributed to me introducing bugs in the past.